docker-compose.yml

```diff
@@ -7,7 +7,6 @@ services:
       - 9007:9007
     volumes:
       - results:/rest_api/tmp
-      - .:/rest_api
```

Comment:
I'd like this line to be included, so that I can integrate changes to the API more easily with an existing image. If you think it's better, however, I might just change this locally for me.

Reply:
Yes, you should change it locally. Btw, including this line also mounts the .git folder into the container, which should be avoided.